shell/platform/android/io/flutter/plugin/editing/InputConnectionAdaptor.java

```diff
@@ -473,6 +482,65 @@ public boolean performEditorAction(int actionCode) {
     return true;
   }
 
+  @Override
+  public boolean commitContent(InputContentInfo inputContentInfo, int flags, Bundle opts) {
```

Comment:
A good point was recently brought up by @matthew-carroll about how APIs like this should follow Flutter's declarative pattern instead of being imperative.  So instead of telling the client to commit the content, informing the client that the user committed content (`contentCommitted` maybe?).  Though, it seems that adjacent API methods already have an imperative pattern, so maybe now is not the time to change that.